modules/ROOT/partials/configuration/advanced-typography.adoc

```diff
@@ -0,0 +1,205 @@
+[[content_langs]]
+== `+content_langs+`
```

Comment:
You may be thinking of `typography_default_langs` which has this functionality. [`content_langs` already defined](https://github.com/tinymce/tinymce-docs/blob/86adeabb818e5270cc2ce19f5dfd31ff67ed30e8/modules/ROOT/partials/configuration/content_langs.adoc)